src/services/ledger/schema.types.d.ts

```diff
@@ -1,3 +1,44 @@
+interface TransactionRecord {
```

Comment:
if this is is used to match medici we will need to add missing fields

https://github.com/GaloyMoney/galoy/blob/74ee147888b00c21825145c1097ace8aaf5e68d8/src/services/ledger/schema.types.d.ts

Reply:
Oh I didn't realise you all were making the same type. I swapped that new type in here instead.